bestla/bestla/bestla_prologue_b.h

```diff
@@ -486,8 +485,8 @@ class WeightKBlockNInteger {
             for (size_t i = thdp.loc[0]; i < thdp.loc[0] + thdp.size[0]; i++) {
               for (size_t j = thdp.loc[1]; j < thdp.loc[1] + thdp.size[1]; j += 2) {
                 auto src = *(B + i * ldb / 2 + j / 2);
-                s8ptr[(j + 0) * N + i] = ((src & 0xf) - 8) << 4;
-                s8ptr[(j + 1) * N + i] = (((src & 0xf0) >> 4) - 8) << 4;
+                s8ptr[(j + 0) * N + i] = ((src & 0xf) - 8);
```

Comment:
this mean we should modify all repack pre-process codes right? and looks like we should recover the sign when we decompress, why we change to this?

Reply:
Saving high bits can ignore the sign bit, but is hard to remove the zero points. Saving the low bits, we can always get the zero points removed after the decompression, because `(Q - zero point)` is always within the range of int8. BTW, if using zero points, the bias value (`8`) can be added to the zero point.